x-pack/solutions/security/plugins/security_solution/public/detections/components/alert_summary/table/table.tsx

```diff
@@ -0,0 +1,190 @@
+  'xpack.securitySolution.alertSummary.table.column.severity',
+  { defaultMessage: 'Severity' }
+);
+const RULE_NAME_COLUMN = i18n.translate(
+  'xpack.securitySolution.alertSummary.table.column.ruleName',
+  { defaultMessage: 'Rule' }
+);
+
+const TIMESTAMP = '@timestamp';
+const RELATED_INTEGRATION = 'kibana.alert.rule.parameters';
+const SEVERITY = 'kibana.alert.severity';
+const RULE_NAME = 'kibana.alert.rule.name';
+
+const columns: EuiDataGridProps['columns'] = [
+  {
+    id: TIMESTAMP,
+    displayAsText: TIMESTAMP_COLUMN,
+  },
+  {
+    id: RELATED_INTEGRATION,
+    displayAsText: RELATION_INTEGRATION_COLUMN,
+  },
+  {
+    id: SEVERITY,
+    displayAsText: SEVERITY_COLUMN,
+  },
+  {
+    id: RULE_NAME,
+    displayAsText: RULE_NAME_COLUMN,
+  },
+];
+
+const ALERT_TABLE_CONSUMERS: AlertsTableProps['consumers'] = [AlertConsumers.SIEM];
+const RULE_TYPE_IDS = [ESQL_RULE_TYPE_ID, QUERY_RULE_TYPE_ID];
+const ROW_HEIGHTS_OPTIONS = { defaultHeight: 40 };
+const TOOLBAR_VISIBILITY: EuiDataGridToolBarVisibilityOptions = {
+  showDisplaySelector: false,
+  showKeyboardShortcuts: false,
+  showFullScreenSelector: false,
+};
+const GRID_STYLE: EuiDataGridStyle = { border: 'horizontal' };
+
+export interface TableProps {
+  /**
+   * DataView created for the alert summary page
+   */
+  dataView: DataView;
+  /**
+   * Groups filters passed from the GroupedAlertsTable component via the renderChildComponent callback
+   */
+  groupingFilters: Filter[];
+}
+
+/**
+ * Renders the table showing all the alerts. This component leverages the ResponseOps AlertsTable in a similar way that the alerts page does.
+ * The table is used in combination with the GroupedAlertsTable component.
+ */
+export const Table = memo(({ dataView, groupingFilters }: TableProps) => {
+  const {
+    services: {
+      application,
+      data,
+      fieldFormats,
+      http,
+      licensing,
+      notifications,
+      uiSettings,
+      settings,
+    },
+  } = useKibana();
+  const services = useMemo(
+    () => ({
+      data,
+      http,
+      notifications,
+      fieldFormats,
+      application,
+      licensing,
+      settings,
+    }),
+    [application, data, fieldFormats, http, licensing, notifications, settings]
+  );
+
+  const getGlobalFiltersSelector = useMemo(() => inputsSelectors.globalFiltersQuerySelector(), []);
+  const globalFilters = useDeepEqualSelector(getGlobalFiltersSelector);
+
+  const { to, from } = useGlobalTime();
+  const timeRangeFilter = useMemo(() => buildTimeRangeFilter(from, to), [from, to]);
+
+  const filters = useMemo(
+    () => [
+      ...globalFilters,
+      ...timeRangeFilter,
+      ...groupingFilters.filter((filter) => filter.meta.type !== 'custom'),
+    ],
+    [globalFilters, groupingFilters, timeRangeFilter]
+  );
+
+  const dataViewSpec = useMemo(() => dataView.toSpec(), [dataView]);
+
+  const { browserFields } = useMemo(
+    () => getDataViewStateFromIndexFields('', dataViewSpec.fields),
```

Comment:
Is this the only place in the page it's being used? Just want to make sure we're not repeating our current mistake in security of calling this in multiple places 😅 .